libexec/cli/list.help

```diff
@@ -0,0 +1,32 @@
+USAGE: singularity [...] list [list options...]
+
+List all running named instances
```

Comment:
If the name is going to change, (and I like `list` just fine) I would advocate for `status` to be more in line with the daemon mindset (that we are thinking about but avoiding).  😆